### PR TITLE
QOL changes to NSS kerberos

### LINKED ISSUE #20131: Small issues with deltastation
**Issue Description**:
A series of small mapping issues with delta station.
1. Engineering and Research lack a recharger. On every other map, every core department (not including service as that is spread out) contains at least one recharger, either for staff to charge their own equipment/weapons or for security's use when defending such an area from a biohazard.
2. The kitchen lacks shutters over the counters. As both other main maps we use have shutters in the kitchen, this seems like an oversight.
3. The table located INSIDE the kitchen is reinforced, when it should not be. The counters are meant to, as this is consistent across all maps, but the internal tables are just meant to be normal metal ones.
4. Miners have access to the cargo bay, an area they do not normally have access to on the other stations. They are meant to be able to access the cargo office (and I'm assuming the break room area, all circled in red) however they can use the door circled in blue but none of the areas in the area they should be able to get to.
image

**What did you expect to happen**: 
No issues with maps reeeeee bad game 0/10

**Extra information**:
Just small stuff either for QOL or consistency, nothing overly game breaking or urgent

### PR DESCRIPTION
**Pull request by Fluffyzack21**
## What Does This PR Do
 Fixes the access to cargo so miners can't get thru cargo, replaced inside of kitchen to normal tables instead of reinforced tables, added some shutters in the kitchen, added rechargers in departments

 
Fixes #20131 

## Why It's Good For The Game
 Some QOL changes to NSS Kerberos and helps with unwanted people going into places that they should not be in and now you can recharge your gun in any core department, and some other stuff for the looks

## Images of changes
 
Screenshot 2023-12-01 142515
image
Screenshot 2023-12-01 142337
Screenshot 2023-12-01 141831

## Testing

  1. spawned in as a science person and shot a human to death then recharged my gun and made sure the rechargers worked
2. Then spawned myself as a chef and checked if the Table are normal tables inside the kitchen and they were and plus I checked if I could open and close the shutters and they could, and tested the other things
3. Spawned as a miner and check if I had to access to cargobay and I do not

## Changelog
:cl:
add: Added rechargers in engineering and in science, and moved them a little to make it look nicer on NSS Kerberos
add: Added a button for opening and closing the shutters in the kitchen, and tint control for one of the doors on NSS Kerberos
add: Added shutters in the kitchen linked with the button I just added on NSS Kerberos
tweak:  Replaced the reinforced tables to normal tables in the kitchen on NSS Kerberos
add: Added a cyborg recharger in engineering on NSS Kerberos
:cl:

### CODE REVIEW

**Comment by Fluffyzack21** [comment]
Screenshot 2023-12-17 092522

**Comment by Fluffyzack21** [comment]
Screenshot 2023-12-17 100216

**Comment by Aylong220** [comment]
Miners can't go into the cargo, but have access to the rest room
image

You added shutters but the doors are still glass, so they're useless
image

Plus there's no shutters under the windows and left.
image

I'm not sure if it even needs a recharger
image

God, I hate it when DiffBot shows so much more than it actually does.

**Comment by Fluffyzack21** [comment]
> Miners can't go into the cargo, but have access to the rest room image
> 
> You added shutters but the doors are still glass, so they're useless image
> 
> Plus there's no shutters under the windows and left. image
> 
> I'm not sure if it even needs a recharger image
> 
> God, I hate it when DiffBot shows so much more than it actually does.

so I fix the issue with the miners, I was thinking only having the two shudders for the two sides people have access I was really not caring about the windows, and the recharger in service was becuse someone suggested it on discord since other station have it in a service dept

**Comment by Fluffyzack21** [comment]
Screenshot 2024-01-07 230211

**Comment by Fluffyzack21** [comment]
Screenshot 2024-01-07 233019

File: _maps/map_files/Delta/delta.dmm
Comment (warriorstar-orion) [inline _maps/map_files/Delta/delta.dmm:85429]: These shouldn't be changed to mail_sorting, that's meant for the Delivery Office only

Comment (Fluffyzack21) [reply _maps/map_files/Delta/delta.dmm:85429]: so you want me just remove the airlock_helper on that door instead?

Comment (Fluffyzack21) [reply _maps/map_files/Delta/delta.dmm]: fixed

**Comment by warriorstar-orion** [comment]
image

I would suggest leaving this door with its original access. We don't have any hard and fast rules for "should miners have access to the bay" but:

1. On Cyberiad, miners don't have access to the bay,
2. On Cere, miners have access to the bay because there's no airlocks between the dock and the bay,
3. On Meta, miners must move through the bay in order to access the dock.

So the miners having access to the bay here isn't exactly atypical.

The other options are:

1. Keep the airlock but give cargo access to mining, which is atypical; no stations give cargo access to the mining dock
2. Keep the airlock but make it QM only
3. Remove the airlock

**Comment by Fluffyzack21** [comment]
> /obj/effect/mapping_helpers/airlock/access/all/supply/general

The reason for I made for the blue I did mail_sorting is becuse HOP is no longer cargo head since the QM is now a head, and making it that way made it so HOP can't access cargo due to QM now being command and only does service, since cyberiad already got there doors fixed relating to this

**Comment by Fluffyzack21** [comment]
DO NOT MERGE UNTILL #23819 Is merged, once that PR is merged I need to fix the door access

**Comment by Henri215** [comment]
image
Specifiy the station on those

**Comment by Henri215** [comment]
> DO NOT MERGE UNTILL #23819 Is merged, once that PR is merged I need to fix the door access in cargo

For that to happen you will have to request a queue skip then, because that warrior PR is later on the line.

**Comment by Fluffyzack21** [comment]
> > DO NOT MERGE UNTILL #23819 Is merged, once that PR is merged I need to fix the door access in cargo
> 
> For that to happen you will have to request a queue skip then, because that warrior PR is later on the line.

how would I do that?

**Comment by Henri215** [comment]
> > > DO NOT MERGE UNTILL #23819 Is merged, once that PR is merged I need to fix the door access in cargo
> > 
> > 
> > For that to happen you will have to request a queue skip then, because that warrior PR is later on the line.
> 
> how would I do that?

Just have to ask, be it here or discord, someone will see and update it.

**Comment by warriorstar-orion** [comment]
Since this PR is now leading the map queue for Delta, I'd suggest undrafting it and keeping the access changes you've made as is, and I'll circle back to fix them after my new access helpers go in. I'm pretty sure the rest of the changes are fine.

**Review by warriorstar-orion — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**
